src/components/modal.jsx

```diff
@@ -24,13 +21,15 @@ class Modal extends React.Component {
 			return;
 		}
 
-		this.props.onClose()
+		if (this.props.ignoreClose !== true) {
+			this.props.onClose()
+		}
 	};
 
 	componentDidMount() {
 		modalRoot.appendChild( this.element );
 		document.addEventListener('mousedown', this.clickOutsideListener);
-  		document.addEventListener('touchstart', this.clickOutsideListener);
+  	document.addEventListener('touchstart', this.clickOutsideListener);
```

Comment:
mix of spaces and tabs 
```suggestion
		document.addEventListener('touchstart', this.clickOutsideListener);
```